<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteInjecterClass</class>
 <widget class="QMainWindow" name="RemoteInjecterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RemoteInjecter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_targetFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>目标文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_launch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_workingDir">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>40</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>选择路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_targetFile">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_workingDir">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dll">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_workingDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>工作目录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cmdLineArgs">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>621</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_cmdLineArgs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>命令行参数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dll">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>待注入dll</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_targetFile">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dll">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>110</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_log">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>711</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_log">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>log输出</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_childProcess">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>InjectChildProcess</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RemoteInjecter.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_targetFile</sender>
   <signal>clicked()</signal>
   <receiver>RemoteInjecterClass</receiver>
   <slot>SelectTargetFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_workingDir</sender>
   <signal>clicked()</signal>
   <receiver>RemoteInjecterClass</receiver>
   <slot>SelectWorkingDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>758</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_dll</sender>
   <signal>clicked()</signal>
   <receiver>RemoteInjecterClass</receiver>
   <slot>SelectDLL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>759</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_launch</sender>
   <signal>clicked()</signal>
   <receiver>RemoteInjecterClass</receiver>
   <slot>LaunchEXE()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
  <slot>SelectTargetFile()</slot>
  <slot>SelectWorkingDir()</slot>
  <slot>SelectDLL()</slot>
  <slot>LaunchEXE()</slot>
 </slots>
</ui>
